README.md

```diff
@@ -695,8 +770,23 @@ Private properties should be declared in class extensions (anonymous categories)
 @end
 ```
 
+## Private headers
+
+Use private headers to define class extension and any internal methods and properties which can be used for [unit tests](#unit-tests). Also they have to contain all necessary inclusions.
```

Comment:
Yes, the idea it to include everything needed in private header as it's useful for creation of mocks in tests, etc. Perhaps it has to be extended with additional details like: *All includes for forward declarations in header or used types in order to avoid their inclusion in tests and keep them in sync*